src/main.rs

```diff
@@ -72,8 +73,20 @@ fn act_on_file(language: LANG, path: PathBuf, cfg: Config) -> std::io::Result<()
     } else if cfg.metrics {
         let source = read_file_with_eol(&path)?;
         let language = guess_language(&source).unwrap_or(language);
-        let cfg = MetricsCfg { path };
-        action::<Metrics>(&language, source, &cfg.path.clone(), pr, cfg)
+        if cfg.output.is_empty() {
```

Comment:
nit: there is some dup code in then & else clauses (finally the only difference is almost output_path definition in the struct)

Reply:
Fixed